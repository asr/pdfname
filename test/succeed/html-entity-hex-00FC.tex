\documentclass{article}

% The real example is
% https://link.springer.com/article/10.1007/BF01449946.

\pdfinfo{%
  /Author (Begr\&\#x00FC;ndung)%
  /Title  (Begr\&\#x00FC;ndung)%
  /CreationDate (D:2017)%
}

\begin{document}
pdfname test.
\end{document}
