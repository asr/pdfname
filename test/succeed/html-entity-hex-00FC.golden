ret > ExitSuccess
out > The full path name will be /tmp/begrundung-2017.begrundung.pdf
out >
